libsigner/src/signer_set.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) 2024 Stacks Open Internet Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use blockstack_lib::chainstate::stacks::boot::NakamotoSignerEntry;
+use hashbrown::{HashMap, HashSet};
+use stacks_common::types::chainstate::{StacksAddress, StacksPublicKey};
+use wsts::curve::ecdsa;
+use wsts::curve::point::{Compressed, Point};
+use wsts::state_machine::PublicKeys;
+
+/// A reward set parsed into the structures required by WSTS party members and coordinators.
+#[derive(Debug, Clone)]
+pub struct ParsedSignerEntries {
+    /// The signer addresses mapped to signer id
+    pub signer_ids: HashMap<StacksAddress, u32>,
+    /// The signer ids mapped to public key and key ids mapped to public keys
+    pub public_keys: PublicKeys,
+    /// The signer ids mapped to key ids
+    pub signer_key_ids: HashMap<u32, Vec<u32>>,
+    /// The signer ids mapped to wsts public keys
```

Comment:
Wouldn't this be key IDs mapped to WSTS public keys?  Or am I missing something?